tests/psoc6/run_psoc6_tests.sh

```diff
@@ -8,12 +8,35 @@
 echo "executing $0 $* ..."
 
 usage() {
-  echo "Usage: $0 -a <run all meaningful test by directory without excluding any> -c <clean results directory and file before running tests> -d <device to be used> -f <run failing tests only> -i <run implemented tests only and exclude known failing tests> -n <run not yet implemented tests only> -w <run wifi tests => needs secrets.py key file>" 1>&2;
+  echo "Usage:"
+  echo "sh run_psoc6_tests.sh <opt>"
+  echo "Available options:"
+  echo "  -a            run all meaningful test by directory without excluding any"
+  echo "  -c            clean results directory and file before running tests"
+  echo "  -f            run failing tests only"
+  echo "  -i            run implemented tests only and exclude known failing tests"
+  echo "  -n            run not yet implemented tests only"
+  echo "  -w            run wifi tests => needs secrets.py key file>"
+  echo "  --dev0        device to be used"
+  echo "  --dev1        second device to be used (for multi test)"
+  echo "  --psco6       run only psoc6 port related tests"
+  echo "  --psoc6-multi run only psoc6 port multi tests (requires 2 instances)"
   exit 1;
 }
 
+# Converting long arguments into short flags for getopts
+for arg in "$@"; do
```

Comment:
There is a more general way to do this using a more recent version of getopt tools.

Reply:
First time I used the getopts :D Yep, if there is a better, way we can improve it in the next iteration.